pageserver/src/tenant/timeline/compaction.rs

```diff
@@ -819,15 +819,16 @@ pub struct CompactionStatistics {
     time_acquire_lock_secs: f64,
     time_analyze_secs: f64,
     time_download_layer_secs: f64,
+    time_to_first_kv_pair_secs: f64,
     time_main_loop_secs: f64,
     time_final_phase_secs: f64,
     time_total_secs: f64,
 
     // Summary
-    /// Ratio of the key-value size before/after gc-compaction.
-    uncompressed_size_ratio: f64,
-    /// Ratio of the physical size before/after gc-compaction.
-    physical_size_ratio: f64,
+    /// Ratio of the key-value size after/before gc-compaction.
+    uncompressed_size_ratio_inv: f64,
```

Comment:
nit on naming: not super clear what `inv` means. I guess `inverted` ? Why are we inverting it? I guess because `avoid div by 0`. Maybe call it `uncompressed_retention_ratio` and `compressed_retention_ratio` ?